iothub/device/src/TransportSettings/IotHubClientAmqpSettings.cs

```diff
@@ -43,12 +43,6 @@ public IotHubClientAmqpSettings(IotHubClientTransportProtocol transportProtocol
         /// </remarks>
         public TimeSpan IdleTimeout { get; set; } = TimeSpan.FromMinutes(2);
 
-        /// <summary>
```

Comment:
you'll also want to add a remark to the remote cert validation callback calling out that it isn't supported on amqp_ws for this implementation